Revert/Sources/CollectionViewCells.swift

```diff
@@ -25,7 +25,7 @@ class CollectionViewCell: UICollectionViewCell {
     self.titleLabel?.font = UIFont.preferredFontForTextStyle(UIFontTextStyleBody)
     self.subheadLabel?.font = UIFont.preferredFontForTextStyle(UIFontTextStyleSubheadline)
   }
-
+  
```

Comment:
Avoid committing white spaces. I understand this was a big PR, but you should review each line of code before committing them allowing to discard these sort of changes.

No biggies though.